### PR TITLE
Cleaning up tile controls

### PR DESCRIPTION
**Pull request by cquiroz**
Simplifies a bit the tile code and make the controls in different style consistest across each other and when minimized/normal

### CODE REVIEW

File: explore/src/main/scala/explore/targeteditor/AsterismEditor.scala [hunk 1]
```diff
@@ -108,19 +108,19 @@ object AsterismEditor extends AsterismModifier:
 
         <.div(
           ExploreStyles.AladinFullScreen.when(fullScreen.get.value),
-          if (props.readonly) EmptyVdom
-          else
```
Comment (toddburnside) [inline explore/src/main/scala/explore/targeteditor/AsterismEditor.scala:112]: Why do we want to display the target selection popup if they can't do anything with it?

Comment (cquiroz) [reply explore/src/main/scala/explore/targeteditor/AsterismEditor.scala:112]: It makes the layout simpler if we always know there is an element.
But yes we could hide it as well and tweak the css a bit more

Comment (toddburnside) [reply explore/src/main/scala/explore/targeteditor/AsterismEditor.scala:112]: Not a big deal, I guess. Keeping CSS simpler is a good goal.

**Review by toddburnside — APPROVED** [review]
LGTM

File: explore/src/main/scala/explore/targeteditor/AsterismEditor.scala (hunk 1 above)
Comment (toddburnside) [reply explore/src/main/scala/explore/targeteditor/AsterismEditor.scala:112]: Would an empty `span` work as well as the button for keeping the CSS simple?

**Pull request merged**